src/Microsoft.Health.Fhir.SqlServer/Features/Storage/ResourceDateKey.cs

```diff
@@ -0,0 +1,77 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Persistence
+{
+    public class ResourceDateKey : IEquatable<ResourceDateKey>
```

Comment:
Should it be "ResourceDat**a**Key" instead of "ResourceDat**e**Key"?

Reply:
No. Date stresses time component.